shell/client/shell.html

```diff
@@ -183,6 +176,28 @@
   </ul>
 </template>
 
+<template name="notificationItem">
+  <div class="notification-item">
+    <div class="notification-header">
+      {{#if isAppUpdates}}
+        <span>App Market has updated apps</span>
+        <span title="Ignore the below updates" class="cancel-notification">Dismiss</span>
+        <span title="Update all of the below apps" class="accept-notification">Apply Updates</span>
+      {{else}}
+        <span><em>{{grainTitle}}</em> is backgrounded</span>
+        <span title="Stops the background app" class="cancel-notification">Cancel</span>
+      {{/if}}
+      <br>
```

Comment:
Instead of using a `<br>` in HTML, you can make `notification-timestamp` be `display: block` to control this in CSS.